local/network.go

```diff
@@ -226,15 +228,16 @@ func newNetwork(
 
 	// Create the network
 	net := &localNetwork{
-		networkID:          networkID,
-		genesis:            []byte(networkConfig.Genesis),
-		nodes:              map[string]*localNode{},
-		closedOnStopCh:     make(chan struct{}),
-		log:                log,
-		bootstraps:         beacon.NewSet(),
-		newAPIClientF:      newAPIClientF,
-		nodeProcessCreator: nodeProcessCreator,
-		flags:              networkConfig.Flags,
+		networkID:            networkID,
+		genesis:              []byte(networkConfig.Genesis),
+		nodes:                map[string]*localNode{},
+		closedOnStopCh:       make(chan struct{}),
+		log:                  log,
+		bootstraps:           beacon.NewSet(),
+		newAPIClientF:        newAPIClientF,
+		nodeProcessCreator:   nodeProcessCreator,
+		flags:                networkConfig.Flags,
+		unexpectedNodeStopCh: make(chan string),
```

Comment:
This channel isn't buffered, so if someone isn't reading off this channel we're going to block a bunch of goroutines (namely those started in `Start()`) from exiting

Reply:
that's true. Added UnexpectedNodeStopMsgBuffSize to prevent that for a large number of msgs.